### ISSUE TITLE
GT++ Quest tab

### ISSUE THREAD

**Issue by richardhendricks** [richardhendricks/GT5-Unofficial#30]
# High priority quests - all completed
* Alloy Blast Smelter
* Power Sub-Station
* Wireless Charger - will charge any EU item on you, will do tier or lower devices, including tools and armor
* Large sifter
* Maceration stack controller
* Ore washing plant
* Large thermal refinery

# Mid priority quests
* Fishing Port
* Industrial Centrifuge
* Industrial Electrolyzer
* Thorium Reactor
* Multi-Machine
* Pollution Scrubbers
* Multi-tank
* Industrial wire factory
* Industrial Extruder
* Cutting Factory
* semi-fluid generators - for early game
* advanced boilers - for early game
* simple washers - for early game
* energy buffer - prelude to Power Sub-Station

# Low priority quests
* Matter fabricator
* steam condensors
* player safe - should be higher?
* chemical dehydrator
* Industrial coke oven
* Industrial material press
* Rocket engine - should be higher?
* auto-assembler
* Thermal boiler
* radioisotope thermal generator

# Will not do
* Tree farmer - disabled for now

**Comment by draknyte1** [comment]
Energy Buffers/Power Sub-Station now have a book as of yesterday. Power storage should also just be bundled into one quest, with say 2 steps. One done in mv/hv, the next to make the PSS in EV/whenever.

advanced boilers, semi-fluid generators, simple washers - Sorta early game
Thermal boiler - Sorta early Multiblock

Tree Farmer - disabled? - Correcto.

**Comment by richardhendricks** [comment]
I have some quest names/descriptions ready.

"name": "Mallard Rust Smelly",
"description": "Or is it Alloy Blast Smelter? Well, that makes more sense. This multibock creates the higher tier alloys needed for the more advanced GT++ multiblocks. It directly smelts into liquids.",

"name": "Cough cough - Something needs to be done about the smell",
"description": "Birds falling out of the sky, fish dead and floating belly up in the rivers, acid rain melting the ground...Time to start picking up the trash and cleaning up the air.       ",

"name": "EU packets flowing everywhere...",
"description": "...in my dreams. What if instead of having battery packs on every generator, we could centralize all the generation in one location? Put it away from the well travelled parts of the the base? Out in the country somewhere?  Let the animals deal with the pollution.  Anyways, let\u0027s start working on a Power Sub-Station to store EU and deliver it to our base.              ",

"name": "Who cares about a little cancer...",
"description": "...when it means you don\u0027t have to strip naked to recharge your nanosuit? The Wireless Charger will keep your armor and tools charged up without having to put them into a battery buffer. You didn\u0027t want kids anyways...     ",

"name": "Shake that booty...",
"description": "...out of the purified ores.  Anyways, a Large Sifter will increase your sifting rate significantly. What will you do with all these gems??",

"name": "Hulk Smash!",
"description": "Mass destruction! Or at least, lots and lots of crushed ores. Build a maceration statck to keep up with all the ores you are getting.",

"name": "The rocks in the washer...",
"description": "...go round and round, all the live long day. A large orewashing plant gets your rocks nice and clean.",

"name": "Dancing in circles",
"description": "Spinning around, getting hot - sounds like a dance. Or a thermal centrifuge.  Build a large thermal refinery to get the most out of your ores.",

**Comment by richardhendricks** [comment]
What do you mean, bundled in two steps?  Do you mean a quest for energy buffers in MV/HV and the PSS in EV?

The high/mid/low isn't about tier, it is about which ones need to be completed first.

**Comment by richardhendricks** [comment]
ok, re-arranged the priority a little.

**Comment by draknyte1** [comment]
I likey.

Was just thinking that the PSS quest should sit behind the Energy buffer quest, but also be tied to something from EV (Can you even do that? I don't know lul)

**Comment by richardhendricks** [comment]
yeah sure that's possible.  I am wondering if we shouldn't put the GT++ quests spread around.  I am going to check if we can do both.  It would be nice to see them in the tiers that matter.  IE, GT++ pollution scrubber after EBF.

**Comment by richardhendricks** [comment]
Want to open a sushi bar?
Then you'll need lots of fish.  The Fishing Port is a multiblock that will increase your piscatorial output.  Also great for making biodiesel.
Need to gate behind planet with source for alloy metals.

You spin my head...
...right round.  Tired of waiting for your lowly single block centrifuges?  Time to make a multiblock!

Got a Superman infestation?
Thorium reactors make great power sources, producing 4A of IV once they have warmed up. They can also produce useful byproducts like Krypton. And with 80 perecent less chance of accidental explosions vs the IC2 reactors, what's not to love?
Just call it 'Nuclear chemistry is a bit..' "Of this and that, truly. Gaze upon the horrors of the chemical processing chain for 'name of a molten salt', this will be your goal if you wish to produce heavy fluorides and other rare materials au h as krypton. "
'Quest title'
"Desc"


Insert Tab A into Slot B
The multi-machine lets you emulate the less popular GT single block machines. It even comes with its own manual!
In game manual

Tanks for the Memories
Ready to change back to multiblock tanks?  You can even store more than one fluid in this Multitank. (capacity)
(reqs)

Digi-key Direct Catalog Sales!
Well, not really, but when you need massive rolls of wires and cables, the industrial wire factory is the way to roll.

It's like Play-Doh for Adults
You put the ingots in, and they squirt out as other things! So much fun!

Keep hands and feet outside the red lines at all times
Or else they might be cut off. The Cutting Factory lets you scale up your slicing and dicing to industrial levels. All the planks and bolts you could need!

Burn baby burn
(semifluid gens)

Rub a dub dub
At least you're not cleaning these by hand in a tub.  Who needs lame byproducts anyways? These are faster and get you the key product right away.

Better than batteries
Tired of filling your battery buffers with rows of batteries?  An energy buffer isn't portable, but at least you don't have to spend hours making a dozen batteries. (how do they work? input/output face? crafting key steps?)
